frontend/src/redux/action.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export const userLoginFetch = (username, password) => {
+  return dispatch => {
+    return fetch("http://localhost:5000/api/login", {
```

Comment:
Can we make the URL a global variable? @jshin029

Reply:
@ericong18 Hey Eric the updates have been made